tools/src/main/java/org/apache/kafka/trogdor/workload/CreateTopicsWorker.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kafka.trogdor.workload;
+
+import org.apache.kafka.clients.admin.AdminClient;
+import org.apache.kafka.clients.admin.AdminClientConfig;
+import org.apache.kafka.clients.admin.NewTopic;
+import org.apache.kafka.common.internals.KafkaFutureImpl;
+import org.apache.kafka.trogdor.common.Platform;
+import org.apache.kafka.trogdor.common.WorkerUtils;
+import org.apache.kafka.trogdor.task.TaskWorker;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+import java.util.concurrent.Future;
+import java.util.concurrent.atomic.AtomicReference;
+
+public class CreateTopicsWorker implements TaskWorker {
+    private static final Logger log = LoggerFactory.getLogger(CreateTopicsWorker.class);
+
+    private static final int MAX_BATCH_SIZE = 10;
+
+    private final String id;
+
+    private final CreateTopicsSpec spec;
+
+    public CreateTopicsWorker(String id, CreateTopicsSpec spec) {
+        this.id = id;
+        this.spec = spec;
+    }
+
+    @Override
+    public void start(Platform platform, AtomicReference<String> status,
+                      KafkaFutureImpl<String> doneFuture) throws Exception {
+        log.info("{}: starting CreateTopicsWorker({}).", id, spec);
+
+        Properties props = new Properties();
+        props.put(AdminClientConfig.BOOTSTRAP_SERVERS_CONFIG, spec.bootstrapServers());
+        try (AdminClient adminClient = AdminClient.create(props)) {
+            List<String> topicsToCreate = new ArrayList<>();
+            for (int i = 0; i < spec.numTopics(); i++) {
+                topicsToCreate.add(WorkerUtils.topicIndexToName(i));
```

Comment:
No, some tests should create topics over time.  But I think this is fine for round trip test.

Reply:
I asked because there is no base name or starting index for the topic. Since this code is not specific to a test, perhaps, it ought to have either a base name or the index to start from?